app/code/Magento/Bundle/Ui/DataProvider/Product/Form/Modifier/BundlePanel.php

```diff
@@ -525,7 +525,8 @@ protected function getOptionInfo()
                         'componentType' => Container::NAME,
                         'component' => 'Magento_Ui/js/form/components/group',
                         'showLabel' => false,
-                        'additionalClasses' => 'admin__field-group-columns admin__control-group-equal',
+                        'additionalClasses' => 'admin__field-group-columns admin__control-group-equal '
```

Comment:
Additional classes can be array, you don't need to concatenate strings

Reply:
fixed